Student/Student/Submissions/SubmissionRubric/RubricViewController.storyboard

```diff
@@ -46,23 +46,65 @@
+                                    <constraint firstItem="vKN-3v-yJh" firstAttribute="top" secondItem="RXI-be-h0l" secondAttribute="bottom" constant="24" id="7Qy-XO-paM"/>
+                                    <constraint firstItem="vKN-3v-yJh" firstAttribute="centerX" secondItem="ls2-gZ-h3j" secondAttribute="centerX" id="fTy-a0-MpT"/>
+                                    <constraint firstItem="RXI-be-h0l" firstAttribute="top" secondItem="ls2-gZ-h3j" secondAttribute="top" id="hLW-7V-fV1"/>
+                                    <constraint firstItem="RXI-be-h0l" firstAttribute="centerX" secondItem="ls2-gZ-h3j" secondAttribute="centerX" id="wuR-Hd-YNF"/>
+                                </constraints>
+                            </view>
                         </subviews>
                         <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
                         <constraints>
                             <constraint firstItem="YpA-cs-tZJ" firstAttribute="bottom" secondItem="MNC-v2-gd3" secondAttribute="bottom" id="2hh-A1-Rh8"/>
                             <constraint firstItem="MNC-v2-gd3" firstAttribute="top" secondItem="YpA-cs-tZJ" secondAttribute="top" id="K43-Tw-gZi"/>
+                            <constraint firstItem="ls2-gZ-h3j" firstAttribute="centerY" secondItem="w0K-cg-DHX" secondAttribute="centerY" id="aP2-X5-brc"/>
+                            <constraint firstItem="YpA-cs-tZJ" firstAttribute="trailing" secondItem="ls2-gZ-h3j" secondAttribute="trailing" id="oW4-fw-6kg"/>
                             <constraint firstItem="YpA-cs-tZJ" firstAttribute="trailing" secondItem="MNC-v2-gd3" secondAttribute="trailing" id="oq9-dG-ItQ"/>
                             <constraint firstItem="MNC-v2-gd3" firstAttribute="leading" secondItem="YpA-cs-tZJ" secondAttribute="leading" id="ozF-kB-RUV"/>
+                            <constraint firstItem="ls2-gZ-h3j" firstAttribute="leading" secondItem="YpA-cs-tZJ" secondAttribute="leading" id="tOS-FF-M5l"/>
                         </constraints>
                         <viewLayoutGuide key="safeArea" id="YpA-cs-tZJ"/>
                     </view>
                     <connections>
                         <outlet property="collectionView" destination="MNC-v2-gd3" id="d5Y-du-RTT"/>
+                        <outlet property="emptyView" destination="ls2-gZ-h3j" id="Vp2-6w-VTB"/>
+                        <outlet property="emptyViewLabel" destination="vKN-3v-yJh" id="SfC-U7-zma"/>
                     </connections>
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="reu-i7-pSx" userLabel="First Responder" sceneMemberID="firstResponder"/>
             </objects>
             <point key="canvasLocation" x="293" y="-210"/>
         </scene>
     </scenes>
+    <resources>
+        <image name="rubricLine" width="24" height="24"/>
```

Comment:
this is likely going to cause a runtime warning, since it's in Core, and not in Student.